### PR TITLE
feature: routesrv enable tracing

### PR DESCRIPTION
**Pull request by szuecs**
feature: routesrv enable tracing

### CODE REVIEW

File: cluster/manifests/skipper/deployment.yaml [hunk 1]
```diff
@@ -509,6 +509,32 @@ spec:
...
+            max-log-value-len={{ .ConfigItems.skipper_ingress_lightstep_max_log_value_len }}
+            max-logs-per-span={{ .ConfigItems.skipper_ingress_lightstep_max_logs_per_span }}
+            propagators={{ .ConfigItems.skipper_ingress_lightstep_propagators }}
+            {{ .Cluster.ConfigItems.skipper_ingress_lightstep_log_events }}
+        env:
+        - name: LIGHTSTEP_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: skipper-ingress
+              key: lightstep-token
```
Comment (AlexanderYastrebov) [inline cluster/manifests/skipper/deployment.yaml:537]: I assume this is 1-1 copy from skipper-ingress, isn't it?

**Comment by AlexanderYastrebov** [comment]
:+1:

File: cluster/manifests/skipper/deployment.yaml (hunk 1 above)
Comment (szuecs) [reply cluster/manifests/skipper/deployment.yaml:537]: more or less, but changed the skipper-ingress -> routesrv where it makes sense

**Comment by szuecs** [comment]
:+1:

**Pull request merged**